app/views/moab_storage/show.html.erb

```diff
@@ -0,0 +1 @@
+<h1> Test </h1>
```

Comment:
This shouldn't be here -- we have no views.  If someone wants this locally, they can add it to .gitignore

Reply:
yeah, not sure how this got added, will remove.